lib/cmds/update.ts

```diff
@@ -74,12 +74,16 @@ function update(options: Options): Promise<void> {
   let gecko = options[Opt.GECKO].getBoolean();
   let ie: boolean = false;
   let ie32: boolean = false;
```

Comment:
Why two variables if they're just gonna be the same thing anyway?  Combine both into `ie32`